selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/WorkspaceOverview.java

```diff
@@ -87,6 +109,119 @@ public void checkNameWorkspace(String nameWorkspace) {
             visibilityOfElementLocated(By.xpath(format(Locators.WORKSPACE_TITLE, nameWorkspace))));
   }
 
+  public void waitNameFieldValue(String workspaceName) {
+    seleniumWebDriverHelper.waitSuccessCondition(
+        driver ->
+            seleniumWebDriverHelper
+                .waitVisibilityAndGetValue(nameWorkspaceInput)
+                .equals(workspaceName));
+  }
+
+  public void waitUntillNoErrorsDisplayed() {
+    seleniumWebDriverHelper.waitAttributeEqualsTo(nameWorkspaceInput, "aria-invalid", "false");
+  }
+
+  public void waitErrorBorderOfNameField() {
+    seleniumWebDriverHelper.waitAttributeEqualsTo(nameWorkspaceInput, "aria-invalid", "true");
+  }
+
+  public void waitSaveButton() {
+    seleniumWebDriverHelper.waitVisibility(By.xpath(SAVE_BUTTON_XPATH));
+  }
+
+  public void waitDisabledSaveButton() {
+    seleniumWebDriverHelper.waitAttributeEqualsTo(
+        By.xpath(SAVE_BUTTON_XPATH), "aria-disabled", "true");
+  }
+
+  public void waitEnabledSaveButton() {
+    seleniumWebDriverHelper.waitAttributeEqualsTo(
+        By.xpath(SAVE_BUTTON_XPATH), "aria-disabled", "false");
+  }
+
+  public void waitExportWsFormOpened() {
+    seleniumWebDriverHelper.waitVisibility(By.xpath(EXPORT_WS_FORM_XPATH));
+  }
+
+  public String getAsFileFormText() {
+    return seleniumWebDriverHelper.waitVisibilityAndGetText(By.xpath(AS_FILE_CONFIG_BODY_XPATH));
+  }
+
+  public void waitConfiguration(String... expectedConfiguration) {
+    asList(expectedConfiguration).forEach(config -> getAsFileFormText().contains(config));
+  }
+
+  public void clickOnAsFileButton() {
+    seleniumWebDriverHelper.waitAndClick(By.xpath(AS_FILE_BUTTON_XPATH));
+  }
+
+  public void waitExportWorkspaceFormOpened() {
+    seleniumWebDriverHelper.waitVisibility(By.xpath(EXPORT_WS_FORM_XPATH));
+    testWebElementRenderChecker.waitElementIsRendered(By.xpath(EXPORT_WS_FORM_XPATH));
+  }
+
+  public void waitExportWorkspaceFormClosed() {
+    seleniumWebDriverHelper.waitInvisibility(By.xpath(EXPORT_WS_FORM_XPATH));
+  }
+
+  public boolean isAsFileTabOpened() {
+    return seleniumWebDriverHelper
+        .waitVisibilityAndGetAttribute(
+            By.xpath(AS_FILE_BUTTON_XPATH + "/parent::md-tab-item"), "aria-selected")
```

Comment:
It's not quite clear which element xpath `AS_FILE_BUTTON_XPATH + "/parent::md-tab-item"` is linked to.